editoast/src/models/rolling_stock/mod.rs

```diff
@@ -134,6 +134,17 @@ impl RollingStockModel {
             Err(e) => Err(e.into()),
         }
     }
+
+    pub async fn delete_by_name(conn: &mut PgConnection, rs_name: String) -> Result<()> {
+        use crate::tables::rolling_stock::dsl::*;
+        match delete(rolling_stock.filter(name.eq(rs_name)))
+            .execute(conn)
+            .await
+        {
+            Ok(_) => Ok(()),
+            Err(e) => Err(e.into()),
+        }
+    }
```

Comment:
I don't think this method is needed, for your use case you already have access to the rolling stock's id